api/children/childrenMiddleware.js

```diff
@@ -1,6 +1,6 @@
 const Children = require('./childrenModel');
 const checkChildExist = async (req, res, next) => {
-  const id = req.params.id;
+  const id = req.params.child_id;
```

Comment:
Good idea changing the id names, it's good to specify where exactly the id comes from